src/js/utils.js

```diff
@@ -0,0 +1,59 @@
+export function capitalizeFirstLetter(string) {
+	return string[0].toUpperCase().concat(string.slice(1));
+}
+
+export function getID(url) {
+	const numbersRegex = /\d/g;
+	return url.match(numbersRegex).slice(1).join('');
+}
+
+// export function eggGroupsToString(obj) {
+//     return obj.map(innerObj => parseFromHyphen(innerObj.name)).join(', ');
+// }
+
+// export function abilitiesToString(obj) {
+//     return obj.map(innerObj => parseFromHyphen(innerObj.ability.name)).join(', ');
+// }
+
+function parseFromHyphen(string) {
+	return capitalizeFirstLetter(string.replaceAll('-', ' '));
+}
+
+export function objectPropsToString(obj, prop) {
```

Comment:
The infinitive verb is missing here. What's obj and prop?

Reply:
This function it's kinda sketchy, i know. It was meant to replace two separate functions extracting a property of an object, with the only difference being the property depth inside the object